src/components/graphs/assets/bargraphAssets.js

```diff
@@ -0,0 +1,288 @@
+const states = [
+  {
+    name: 'Alabama',
+    abbreviation: 'AL',
+  },
+  {
+    name: 'Alaska',
+    abbreviation: 'AK',
+  },
+  {
+    name: 'American Samoa',
```

Comment:
We should evaluate if our data includes any incidents from places that aren't officially states like Guam and Puerto Rico. If so, they shouldn't be considered in the point selection.

Reply:
Sounds good, I wanted to be a little forward thinking just in case we did get data from there at some point in the future.

I can remove those now

Reply:
Removed